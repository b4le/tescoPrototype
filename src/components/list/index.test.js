import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import List from './index.jsx';
import renderer from 'react-test-renderer';

describe('List Component', () => {
    it('renders correctly', () => {
        const mockProps = {
            params: {
                listId: "1"
            }
        }
        const list = renderer.create(<MemoryRouter><List match={mockProps}/></MemoryRouter>).toJSON();
        expect(list).toMatchSnapshot();
    });
});