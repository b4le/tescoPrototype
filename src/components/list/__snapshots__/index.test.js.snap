// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Component renders correctly 1`] = `
<div>
  <a
    className="sc-bdVaJa hYXBqG"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <h1
    className="sc-bwzfXH kavGrA"
  >
    1
     List here!
  </h1>
</div>
`;
